ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -869,6 +885,66 @@ void OwInterface::guardedMoveAction (double x, double y, double z,
      guarded_move_done_cb<GuardedMoveResultConstPtr> (Op_GuardedMove));
 }
 
+void OwInterface::armMoveJoint (bool relative,
+                                int joint, double angle,
+                                int id)
+{
+  if (! markOperationRunning (Op_ArmMoveJoint, id)) return;
+  thread action_thread (&OwInterface::armMoveJointAction,
+                        this, relative, joint, angle, id);
+  action_thread.detach();
+}
+
+void OwInterface::armMoveJointAction (bool relative,
+                                      int joint, double angle,
+                                      int id)
+{
+  ArmMoveJointGoal goal;
+  goal.relative = relative;
+  goal.joint = joint;
```

Comment:
Subtle problem here.  goal.joint is really an int64 (long int).  The assignment is safe since no precision is lost, but I added a comment.